libs/media-messages/src/generated/types/payloads/media/commands/check-finish-ingest-item-command.ts

```diff
@@ -3,7 +3,7 @@
  */
 export interface CheckFinishIngestItemCommand {
   /**
-   * A unique identifier that is used to track ingest messages for a single document to keep ingest document status up-to-date.
+   * String of minimum length of 1 character, which is also cannot consist of only whitespace characters.
```

Comment:
Here and further down we should keep the original message as it is much more helpful.

Reply:
Related to Ruwans comment: https://github.com/Axinom/mosaic-media-template/pull/98#pullrequestreview-1500138990